### ISSUE TITLE
Floodlight Cam Support

### ISSUE THREAD

**Issue by mariusstrom** [birkir/homebridge-plugin-eufy-security#2]
**Is your feature request related to a problem? Please describe:**
I wish my Eufy wired doorbell and floodlight cameras were supported in HomeKit. :)

**Describe the solution you'd like:**
Floodlight cameras working in HomeKit. Bonus points if light can be exposed as a switch to HomeKit as well.

**Describe alternatives you've considered:**
Buying different floodlights that work with HomeKit... Ugh.

**Comment by birkir** [comment]
If I can get video stream to work in general, I can support more video cameras from eufy.

I can also add the light as a button, sure thing 😀

**Comment by mariusstrom** [comment]
@birkir yep, I saw your note in README.md that it's "for now" a doorbell only plugin, and I assumed that once video stream for one class of Eufy device was functional then the others may not be far behind. 👍

**Comment by birkir** [comment]
Good news, I got video to work (with audio).

I have the 2C cameras I can test with. I'll hopefully have something by tomorrow.

**Comment by a-iac** [comment]
Excellent work man! I agree with @mariusstrom that possibility for other eufy products to Homekit would be amazing. I've got motion sensors and door/window sensors that would be ideal to be added to homekit. Any chance for those? Thank you.

**Comment by birkir** [comment]
It's very hard to program for something you don't have 😁

I would either need to get the products or have someone that has the products help me.

**Comment by a-iac** [comment]
True. I could try, anyway I can of course.

Lead the way! 😃

**Comment by birkir** [comment]
I would need to get a dump of the logs ("found device X, and all the events "push message: ")

I need to add more logs for the parameters details.

**Comment by a-iac** [comment]
How can I acquire the logs?

**Comment by mariusstrom** [comment]
@birkir if you're in the USA (or a country with 110V power), I would strongly consider gifting you a Eufy Floodlight cam if that's the impediment to progress on this.

**Comment by birkir** [comment]
I think I've got enough logs in there for you guys to help me out through github (use version 0.1.3, homebridge -D)

If you can give me a dump of 
1. homebridge startup (device discovery)
2.push messages when turning things on/off through the eufy app (light, etc.)

**Comment by mariusstrom** [comment]
is there any way to get these logs from HOOBS? I've got the plugin working great but there's nothing in the logs for discovery that mentions by floodlight cameras.

**Comment by KrazyKatnip** [comment]
Is there any update regarding this feature?
